realworld-OOP-lab/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "OOP-challenge",
+  "version": "1.0.0",
+  "description": "realworld-OOP-lab folder",
+  "main": "src/OOP-challenge.js",
+  "directories": {
+    "test": "tests"
+  },
+  "scripts": {
+    "test": "mocha --compilers js:babel-core/register ./tests/tests.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "Andel-Bootcamp"
```

Comment:
This should be your repository URL

Reply:
okay. seen. I would implement this on subsequent challenges